pytket/tests/mitigation_test.py

```diff
@@ -119,12 +119,9 @@ def test_spam_integration() -> None:
     def check_correction(
         counts0: dict[tuple[int, ...], int], counts1: dict[tuple[int, ...], int]
     ) -> bool:
-        if (
-            counts0[(0, 0, 0)] > counts1[(0, 0, 0)]
-            and counts0[(1, 0, 1)] > counts1[(1, 0, 1)]
-        ):
-            return True
-        return False
+        return bool(
```

Comment:
Why do we need the cast to `bool` here? That looks like the type we have already.

Reply:
done